Source/capture.cpp

```diff
@@ -130,18 +131,18 @@ static bool CapturePix(WORD width, WORD height, WORD stride, BYTE *pixels, std::
 /**
  * Returns a pointer because in GCC < 5 ofstream itself is not moveable due to a bug.
  */
-static std::ofstream *CaptureFile(char *dst_path)
+static std::ofstream *CaptureFile(std::string *dst_path)
 {
-	char path[MAX_PATH];
-
-	GetPrefPath(path, MAX_PATH);
-
-	for (int i = 0; i <= 99; i++) {
-		snprintf(dst_path, MAX_PATH, "%sscreen%02d.PCX", path, i);
-		if (!FileExists(dst_path))
-			return new std::ofstream(dst_path, std::ios::binary | std::ios::trunc);
+	*dst_path = GetPrefPath() + "screen00.PCX";
+	char &d0 = (*dst_path)[dst_path->size() - 6];
+	char &d1 = (*dst_path)[dst_path->size() - 5];
+	for (int i = 0; i <= 99; ++i) {
+		d0 = '0' + (i / 10);
+		d1 = '0' + (i % 10);

```

Comment:
a bit complicated/magic compared to sprintf

Reply:
Want me to change to sprintf for the filename part?

Reply:
I would prefer that yes.

Reply:
Done